version/version.go

```diff
@@ -0,0 +1,14 @@
+package version
```

Comment:
can you reuse the verision package from sm? move it in pkg/version and log "Application" instead so that it can be reused from different components